services/datalad/datalad_service/common/annex.py

```diff
@@ -50,9 +50,9 @@ def read_ls_tree_line(gitTreeLine, files, symlinkFilenames, symlinkObjects):
     filename, mode, obj_type, obj_hash, size = parse_ls_tree_line(
         gitTreeLine)
     # Skip git / datalad files
-    if filename.startswith('.git/'):
+    if filename.startswith('.git'):
         return
-    if filename.startswith('.datalad/'):
+    if filename.startswith('.datalad'):
         return
     if filename == '.gitattributes':
         return
```

Comment:
This will now be covered by L53. Are we happy to also be skipping `.gitmodules`, `.gitignore`, etc?

Reply:
This just affects if OpenNeuro displays these, so do we want to display .gitignore? We don't allow upload of it but it could exist via git push.

Reply:
I think it's fine to exclude from display. We should probably reject git pushes with gitignores until we drop the working tree.